src/toolkits/object_detection/object_detector.cpp

```diff
@@ -539,6 +554,132 @@ variant_map_type object_detector::perform_evaluation(gl_sframe data,
   return result_map;
 }
 
+std::unique_ptr<model_spec> object_detector::init_darknet_yolo() const {
+
+  std::unique_ptr<model_spec> nn_spec(new model_spec);
+
+  // Initialize a random number generator for weight initialization.
+  std::seed_seq seed_seq = { read_state<int>("random_seed") };
+  std::mt19937 random_engine(seed_seq);
+
+  // Initilize Layer 0
+  int num_features = 3;
+  xavier_weight_initializer conv0_init_fn(16*3*3, 16*3*3, &random_engine);
```

Comment:
Should some of the `3`s on the next several lines be replaced with `num_features`?

Reply:
'num_features' refers to kernel channels here, not image channels so I didn't want to mix them up by replacing the '3's in num of image channels.